topics/admin/tutorials/monitoring-maintenance/slides.html

```diff
@@ -28,37 +28,30 @@
 
 Has a lot of tools and lists that are useful for administrators of the Galaxy server.
 
-* Tools that list:
   * Data types
   * Data tables
   * Display applications
-
-
-* Tools for manipulating
-  * Users, groups, roles and quotas etc
+  * Users, groups, roles, and quotas
   * Running jobs
   * Installed Tools
-  * Errors etc
+  * Tool whitelist
+  * Tool errors
 
 ---
 class: left
 ## Data Types
 
-Data types are Galaxy's method of understanding how to handle data.
-
-* Display of that data
-* Which tools to present the data to
-* Mime-types etc.
-* Whether to allow specification at upload time
-* Which tools are used to convert to different types/versions/indexes
+* Galaxy's method of understanding users' files.
+* Data types and converters are shipped with Galaxy.
```

Comment:
Data types can also be installed from the ToolShed.

Reply:
@nsoranzo I don't want to encourage that

Reply:
I think that in this context it may be useful for a sysadmin (which may be taking over the maintenance or sharing it with others) to known that some installed datatypes can come from the ToolShed. You can mention that's discouraged/deprecated.

My £0.02

Reply:
@nsoranzo I plan to mention it but I do not want to add it to this standalone material, let me add a note

Reply:
added a presenter's note

Reply:
Thanks, works for me!